lc0/src/mcts/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_library (mcts SHARED
```

Comment:
What does SHARED mean here? Will it build a bunch of .so's ?
Maybe it makes sense to make it STATIC (will build.a) or MODULE (I don't know what the last means, but suppose it will not create library at all, but instead feeds all .o directly to linker)

Reply:
Currently it will build a bunch of .so's indeed. I'm not familiar with MODULE, but static doesn't fail with an incorrect dependency tree.

Reply:
Doesn't or does? If doesn't, why not change it? :)